javaslang/src/main/java/javaslang/collection/Tree.java

```diff
@@ -11,12 +11,32 @@
 import javaslang.collection.List.Nil;
 import javaslang.collection.Tree.Empty;
 import javaslang.collection.Tree.Node;
-import javaslang.collection.TreeModule.*;
+import javaslang.collection.TreeModule.FlatMap;
```

Comment:
I analyzed another classes and found out that there is no really any "common denominator" when considering importing strategy, @danieldietrich am I right?